R/assets.R

```diff
@@ -342,7 +342,7 @@ assets_info <- function() {
     installed_versions <- "(None)"
   }
 
-  cat(
+  message(
```

Comment:
We should revert this until we update all messages to either by `{cli}` or `rlang::inform` (or similar)